traveler_ui_kit/build.gradle

```diff
@@ -38,8 +38,8 @@ android {
 dependencies {
 
     //Traveler SDK
-//    api project(path: ':traveler_core_kit')
-    api "com.guestlogix:traveler_core_kit:${traveler_core_version}"
+    api project(path: ':traveler_core_kit')
+    //api "com.guestlogix:traveler_core_kit:${traveler_core_version}"
```

Comment:
Asked @amirziarati how to correctly reference new traveler_core_kit version here.

Reply:
the path dependency is only for testing. you should push it to server. whenever you changed core you push your changes and then when the changes are merged to master a new core build is ready to use. then you ping me and i push a new ui sdk with updated core version or you can make a branch change the ui core version to latest and send a pr.

Reply:
Ok. 
Splitted this branch into two separate branches.

Now this PR contains only core-sdk changes. When it merged - I'll create second PR for ui-sdk